english/dimensions/dimension.md

```diff
@@ -2,6 +2,8 @@ And here you enter a new [dimension](https://www.youtube.com/watch?v=sea8yNCDqWo
 
 Or decide to play a game of chess [chess](../chess/chess.md).
 
+or would you like to challenge Yami Yugi and [dimension the dice!](../Yu-Gi-Oh/yami%20yugi.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.